packages/flutter/lib/src/material/search.dart

```diff
@@ -188,6 +188,12 @@ abstract class SearchDelegate<T> {
   ///  * [AppBar.leading], the intended use for the return value of this method.
   Widget? buildLeading(BuildContext context);
 
+  /// {@macro flutter.material.appbar.automaticallyImplyLeading}
+  bool? automaticallyImplyLeading;
+
+  /// {@macro flutter.material.appbar.leadingWidth} .
```

Comment:
nit: remove the period at the end of this line.

Reply:
defiantly right sorry, i've fixed